### PR TITLE
UI fixes - view screen - IP, remove notifiction icon from header

### PR DESCRIPTION
**Pull request by Ekta29yadav**

### CODE REVIEW

File: apps/front-end/src/pages/front-end/ag-form/Agform.js [hunk 1]
```diff
@@ -130,9 +130,9 @@ export default function Agform({ userTokenInfo }) {
...
     console.log(
       "Latitude: " +
-      position.coords.latitude +
-      "/n Longitude: " +
-      position.coords.longitude
+        position.coords.latitude +
+        "/n Longitude: " +
+        position.coords.longitude
     );
   };
```
Comment (sagarkoshti1990) [inline apps/front-end/src/pages/front-end/ag-form/Agform.js:137]: This required, please ask the developer

File: apps/front-end/src/pages/front-end/splash/PrerakDuties.js [hunk 2]
```diff
@@ -93,30 +92,38 @@ function PrerakDuties(props) {
...
+              width="320px"
+              height="292px"
+              resizeMode="contain"
               source={imgUrl}
               alt={"Alternate Text "}
               //If key is not given image should not be change
               key={imgUrl}
             />
-            <Box bg="white" p="10">
+            <Box bg="white" width="322px" mb="5" py="5" textAlign="center">
```
Comment (sagarkoshti1990) [inline apps/front-end/src/pages/front-end/splash/PrerakDuties.js:126]: Don't use fix width.

File: apps/front-end/src/pages/front-end/splash/SplashScreen.js [hunk 3]
```diff
@@ -101,21 +101,17 @@ export default function SplashScreen({
   React.useEffect(() => {
     //if location is pressed directly setpage screen2
     if (isBackButton === "SplashScreen") {
-      isBackButton = ""
-      console.log(isBackButton, "1")
+      isBackButton = "";
+      console.log(isBackButton, "1");
```
Comment (sagarkoshti1990) [inline apps/front-end/src/pages/front-end/splash/SplashScreen.js:105]: remove all console log if not use.

Comment (Ekta29yadav) [reply apps/front-end/src/pages/front-end/splash/SplashScreen.js]: done

File: apps/front-end/src/pages/front-end/ag-form/Agform.js (hunk 1 above)
Comment (Ekta29yadav) [reply apps/front-end/src/pages/front-end/ag-form/Agform.js]: removed check with swapnil

File: apps/front-end/src/pages/front-end/splash/PrerakDuties.js (hunk 2 above)
Comment (Ekta29yadav) [reply apps/front-end/src/pages/front-end/splash/PrerakDuties.js]: done

**Pull request merged**